app/models/blacklight_config/search_field_configuration.rb

```diff
@@ -27,7 +27,7 @@ def add_field(field)
     end
 
     def all_field
-      SearchField.new("all_fields", :qf => "all_text_timv")
+      SearchField.new("all_fields", :qf => "title_tesim description_tesim date_tesim all_text_timv")
```

Comment:
You shouldn't need to change this - Solr should copy all stored searchable fields into all_text_timv.